src/modules/common/data/tutorial-explorers.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  pages: [
+    {
+      title: 'Welcome to the WildCam Lab!',
+      image: require('../../../images/tutorial/explorer-tutorial-1.gif'),
+      text: 'The WildCam Lab is a place for you to explore data and images from the WildCam Gorongosa project. You can use the map to explore and download data for your own project, or you can identify images and analyze your own data as part of an assignment.',
+    },
+    {
+      title: 'Explore and download data',
+      image: require('../../../images/tutorial/explorer-tutorial-2.gif'),
+      text: 'The Data page allows you to explore and download data from images that the public has identified on WildCam Gorongosa. Click the “View Filters” button and select to items to filter. Click the download button to download a spreadsheet of the data and share it with your students. Click on an orange circle to see a preview of images from that camera that match your filters.',
+    },
```

Comment:
Typo: "select to items to filter" should probably be "select the items to filter".